src/Uno.UI/UI/Xaml/Controls/WebView/Core/CoreWebView2.cs

```diff
@@ -0,0 +1,213 @@
+#nullable enable
+
+using System;
+using Uno.Extensions;
+using Uno.Foundation.Logging;
+using System.Net.Http;
+using Uno.UI.Xaml.Controls;
+using System.Threading;
+using System.Globalization;
+using Windows.Foundation;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Microsoft.Web.WebView2.Core;
+
+public partial class CoreWebView2
+{
+	internal const string BlankUrl = "about:blank";
+	internal const string DataUriFormatString = "data:text/html;charset=utf-8;base64,{0}";
+	internal static readonly Uri BlankUri = new Uri(BlankUrl);
+
+	private readonly IWebView _owner;
+
+	private bool _scrollEnabled;
+	private INativeWebView? _nativeWebView;
+	internal long _navigationId;
+	private object? _processedSource;
+
+	internal CoreWebView2(IWebView owner)
+	{
+		_owner = owner;
+	}
+
+	/// <summary>
+	/// Gets the CoreWebView2Settings object contains various modifiable
+	/// settings for the running WebView.
+	/// </summary>
+	public CoreWebView2Settings Settings => new();
```

Comment:
This quite weird. For the following code:

```csharp
var coreWebView = new CoreWebView2();
coreWebView.Settings.IsWebMessageEnabled = false;
var isWebMessageEnabled = coreWebView.Settings.IsWebMessageEnabled // I think this will be true, which feels incorrect behavior.
```

This is because every time we access `Settings`, it will instantiate a new `CoreWebView2Settings`.

Probably `public CoreWebView2Settings Settings { get; } = new()` is what we want?

Before making the change, we need to add a test.

Reply:
Nice catch, that was unintentional change.